subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ConfigurationResolver.java

```diff
@@ -34,4 +37,7 @@
      * Must be called using the same result instance as was passed to {@link #resolveGraph(ConfigurationInternal, ResolverResults)}.
      */
     void resolveArtifacts(ConfigurationInternal configuration, ResolverResults results) throws ResolveException;
+
+    List<ArtifactRepository> getRepositories();
```

Comment:
:x: Let's add some javadocs and explain that this method should *only* be called for reporting.

Reply:
done